RadixWallet/Features/AppFeature/Overlay/FullScreenOverlayCoordinator+Reducer.swift

```diff
@@ -16,6 +17,7 @@ public struct FullScreenOverlayCoordinator: Sendable, FeatureReducer {
 	}
 
 	public enum DelegateAction: Sendable, Equatable {
+		case claimWallet(ClaimWallet.DelegateAction)
 		case dismiss
```

Comment:
Is this dismiss action still needed here?

Reply:
Yes, but for other reasons. It is used as the fallback if we can't find any action corresponding to the fullscreen ID in question, similar to how it works in `sendAlertAction`.